Gemfile

```diff
@@ -66,5 +65,14 @@ group :development do
   gem 'yard'
 end
 
+group :test do
+  gem 'coveralls', require: false
+  gem 'fakeredis', require: 'fakeredis/rspec'
+  gem 'rspec-benchmark'
+  gem 'mutant-rspec'
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `mutant-rspec` should appear before `rspec-benchmark`.